src/redux/books/books.js

```diff
@@ -23,7 +24,9 @@ const bookReducer = (state = initialState, action) => {
     case ADD_BOOK:
       return [...state, action.payload];
     case DELETE_BOOK:
-      return [...state.filter((payload) => payload.id !== action.payload.id)];
+      console.log('before:', state);
+      console.log('after:', [...state.filter((book) => book.id !== action.payload.id)]);
```

Comment:
- [ ] Please remove `console.logs` from your code as it makes your work look messy 🙏

Reply:
> * [ ]  Please remove `console.logs` from your code as it makes your work look messy 🙏

Thanks for telling i forgot that ❤️‍🔥